actions/artifacts/Multicall3.json

```diff
@@ -0,0 +1,440 @@
+[
+    {
+        "inputs": [
```

Comment:
I don't think you need the whole ABI, right?

only aggregate3value?

<img width="419" alt="image" src="https://github.com/cowprotocol/tenderly-watch-tower/assets/2352112/cc0fefb3-4d47-4b8c-8b41-a50c325647d7">

Reply:
Trimmed.